### ISSUE TITLE
[BUG] Key window constantly switches to an inactive window

### ISSUE THREAD

**Issue by PokeCaptain** [objective-see/LuLu#77]
I have no idea why LuLu does this, but every time I install it, key windows become inactive within seconds. For example, typing in Safari is a chore as the app becomes inactive while I'm typing, refusing any more keyboard input until I click on the window with the mouse. Once I do so, it becomes inactive again within seconds. This applies to all apps, making it impossible to work with LuLu. Again, I have no idea why a firewall application does this, but the issue only shows up with it installed. If you need any logs, I'll be happy to provide. Just point me in the right direction!

**Comment by GraOrigin** [comment]
Sounds like something is stealing your focus...
Had this issue also a while back - not LuLu related, although
Here's a script which checks for the process/app which currently is 'in focus'
Open terminal -> nano search.py -> (paste following)

`#!/usr/bin/python                                                                                                       

try:
    from AppKit import NSWorkspace
except ImportError:
    print "Can't import AppKit -- maybe you're running python from brew?"
    print "Try running with Apple's /usr/bin/python instead."
    exit(1)

from datetime import datetime
from time import sleep

last_active_name = None
while True:
    active_app = NSWorkspace.sharedWorkspace().activeApplication()
    if active_app['NSApplicationName'] != last_active_name:
        last_active_name = active_app['NSApplicationName']
        print '%s: %s [%s]' % (
            datetime.now().strftime('%Y-%m-%d %H:%M:%S'),
            active_app['NSApplicationName'],
            active_app['NSApplicationPath']
        )
    sleep(1)`

save it ( ctrl + O) & exit (ctrl + X) -> run it via 'python search.py' and see the culprit being displayed!

I bet it isn't even LuLu...

**Comment by acheype** [comment]
I can confirm that bug occured to me. I had the focus lost each time I was tipping in a textbox for few seconds.

In order to determine which process was the responsible, I run a script like the one you gave. And I can confirm you that process was "Lulu Helper".

I rebooted first my mac to see if it was still happened. It was.
After, I killed the manually "Lulu Helper" and Lulu restarted it automatically. Then this bug hasn't occured... that's strange.

I will see if it happens again.
